lms/templates/instructor/instructor_dashboard_2/data_download.html

```diff
@@ -1,9 +1,35 @@
+<%namespace name='static' file='/static_content.html'/>
 <%page args="section_data" expression_filter="h"/>
 <%!
 from django.utils.translation import ugettext as _
 from openedx.core.djangolib.markup import HTML, Text
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
 %>
 
+% if configuration_helpers.get_value('ENABLE_SENSITIVE_DATA_MSG_ON_DASHBOARD', settings.FEATURES.get('ENABLE_SENSITIVE_DATA_MSG_ON_DASHBOARD', True)):
+
+<%static:require_module_async module_name="js/vendor/jquery.leanModal" class_name="leanModal">
+
+$( document ).ready(function() {
```

Comment:
we can use the require pattern instead of waiting for the `jquery.ready` thing

Reply:
https://github.com/edx/edx-platform/blob/7a69d2f771fbb89f2d2420811ca12578a5f2163f/cms/templates/widgets/source-edit.html#L31